src/routes/EResourceEditRoute.js

```diff
@@ -65,9 +65,14 @@ class EResourceEditRoute extends React.Component {
 
   handleSubmit = (pci) => {
     const { history, location, mutator } = this.props;
+    const { coverage, suppressFromDiscovery } = pci;
 
     return mutator.pci
-      .PUT(pci)
+      .PUT({
+        id : pci.id,
```

Comment:
May as well destructure the `id` as well, no?

Reply:
oops yeah!!